### PR TITLE
BUG#558528 - SOAPBINDING ANNOTATION WITH SOAPBINDING.STYLE.RPC MAKES …

### PR DESCRIPTION
**Pull request by rtummuru**
…DEPLOYMENT IMPOSSIBLE

### CODE REVIEW

**Comment by rtummuru** [comment]
The changes are minor. I have not created any test case

**Comment by rfelcman** [comment]
Maybe put test somewhere in `org.eclipse.persistence.testing.moxy.unit.jaxb.compiler`.

File: moxy/org.eclipse.persistence.moxy/src/main/java/org/eclipse/persistence/jaxb/javamodel/Helper.java [hunk 1]
```diff
@@ -317,8 +318,9 @@ public boolean isAnnotationPresent(JavaHasAnnotations element, Class annotationC
      *     JavaClass' raw name
      * 2 - the provided JavaClass' raw name starts with "java."
      * 3 - the provided JavaClass' raw name starts with "javax.", with
-     *     the exception of "javax.xml.ws."
-     *
+     *     the exception of "javax.xml.ws." and "javax.xml.rpc"
+     * Note : jax-rpc should not be considered as built-in type. The API itself is
```
Comment (lukasj) [inline moxy/org.eclipse.persistence.moxy/src/main/java/org/eclipse/persistence/jaxb/javamodel/Helper.java]: remove this note as ALL java/javax packages NOT available in the JDK should be considered as not-builtin

File: moxy/org.eclipse.persistence.moxy/src/test/java/org/eclipse/persistence/testing/moxy/unit/jaxb/compiler/SchemaGeneratorTestCase.java [hunk 2]
```diff
@@ -65,4 +77,51 @@ public void addTransformerToSchema(final @Mocked Helper helper, final @Mocked Pr
 
         Deencapsulation.invoke(schemaGenerator, "addTransformerToSchema", property, typeInfo, typeDefParticle, complexType, schema);
     }
+   
+    @Test
+    public void test(final @Mocked Helper helper) {
```
Comment (rfelcman) [inline moxy/org.eclipse.persistence.moxy/src/test/java/org/eclipse/persistence/testing/moxy/unit/jaxb/compiler/SchemaGeneratorTestCase.java]: Could You please change this method name into something more meaningful. This name is too generic.

**Comment by rfelcman** [comment]
Please update copyright year in main code and test class too and try to sign eclipsefdn/eca if You didn't do it before, or just try it again.

**Comment by rfelcman** [comment]
LGTM

**Pull request merged**